### PR TITLE
Change duration of request of speakers to hours:mins

### LINKED ISSUE #2942: Change counted Requests to speak from minutes to hours
**Current behaviour:**
If you count the requests to speak the time is counted as mm:ss

**Reproduction:**
in a meeting:
1. add yourself as an admin
2. put someone on a LoS
3. start and stop their contribution (wait a few seconds between starting and stopping)
4. click on your user name in the top right corner
5. select "Legal notice  Privacy policy"
6. scroll down to "Requests to speak" and start "Count completed requests to speak"
7. you should see something similar to the picture above

**Wanted behaviour:**
the displayed time should not be counted as mm:ss, but as h:mm

### PR DESCRIPTION
**Pull request by reiterl**
Closes #2942 
Update duration service.

### CODE REVIEW

**Review by Elblinator — APPROVED** [review]
function

File: client/src/app/site/services/duration.service.ts
Comment (bastianjoel) [inline client/src/app/site/services/duration.service.ts:1]: The method description should be updated to `Calculates a given time to a readable string, that contains hours and minutes.`

File: client/src/app/site/services/duration.service.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class DurationService {
         const minutes = `0${Math.floor((duration % 3600) / 60)}`.slice(-2);
         const seconds = `0${Math.floor(duration % 60)}`.slice(-2);
         if (!isNaN(+minutes) && !isNaN(+seconds)) {
```
Comment (bastianjoel) [inline client/src/app/site/services/duration.service.ts:81]: Seconds is not needed anymore. 

```suggestion
        const minutes = `0${Math.floor((duration % 3600) / 60)}`.slice(-2);
        if (!isNaN(+minutes)) {
```

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/site/services/duration.service.ts
Comment (reiterl) [reply client/src/app/site/services/duration.service.ts:1]: done.

File: client/src/app/site/services/duration.service.ts (hunk 1 above)
Comment (reiterl) [reply client/src/app/site/services/duration.service.ts]: done.

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]
function

**Pull request merged**